### PR TITLE
Adds Stackdriver Debugger documentation link to the run config.

### LINKED ISSUE #1066: Stackdriver Debug run configuration should include links to documentation

### PR DESCRIPTION
**Pull request by joaoandremartins**
Fixes #1066

### CODE REVIEW

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudDebugRunConfigurationPanel.form [hunk 1]
```diff
@@ -40,12 +40,12 @@
...
         <properties/>
       </component>
-      <component id="c57c1" class="javax.swing.JLabel">
+      <component id="c57c1" class="javax.swing.JLabel" binding="description">
         <constraints>
           <grid row="1" column="0" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties>
-          <text value="Stackdriver Debugger lets you inspect the state of your application without stopping or slowing it down."/>
+          <text value=""/>
```
Comment (patflynn) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudDebugRunConfigurationPanel.form]: does this element need to be there if it's empty?

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudDebugRunConfigurationPanel.java [hunk 2]
```diff
@@ -30,10 +32,12 @@
   private ProjectSelector elysiumProjectId;
   private JPanel panel;
   private HyperlinkLabel docsLink;
+  private JLabel description;
 
   public CloudDebugRunConfigurationPanel() {
     docsLink.setHyperlinkText("For more details, ", "click here", ".");
```
Comment (patflynn) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudDebugRunConfigurationPanel.java]: the hyperlink text also needs to come from a resource bundle.

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudDebugRunConfigurationPanel.form (hunk 1 above)
Comment (joaoandremartins) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/ui/CloudDebugRunConfigurationPanel.form]: We set its value programatically. I didn't edit this file manually.

**Comment by joaoandremartins** [comment]
PTAL

**Pull request merged**